pkg/ibmvpc/examples/out/explain_out/LBToIksNode_all_vpcs_explain_debug.txt

```diff
@@ -0,0 +1,360 @@
+Explaining connectivity from kube-clusterid:1-8fdd1d0a2ce34deba99d0f885451b1ca to 192.168.4.4 (iks-node[192.168.4.4]) within ky-test-vpc
```

Comment:
Why is the LB not reflected in the name here?

Reply:
here it is the input, as it was got from the user.

Reply:
in case a nodeSet is an input, we only write the user input. also see:
Explaining connectivity from vsi3-ky to vsi1-ky within test-vpc1-ky
I can add the kind here, but the solution will apply for all kind of nodeSets. @adisos ?

Reply:
I think adding the kind of nodeSets can improve the report.